PhysicalApp/app/_layout.tsx

```diff
@@ -64,7 +64,7 @@ function RootLayoutNav() {
     <ThemeProvider value={colorScheme === "dark" ? DarkTheme : DefaultTheme}>
       <Stack>
         {token ? (
-                <Stack.Screen name="(tabs)" options={{ headerShown: false }} />
```

Comment:
Why change the routing here?

Reply:
I keep forgetting to remove this change from committing (lmao), I'll change it back! Thanks